build_modules/build.yaml

```diff
@@ -12,6 +13,7 @@ builders:
         - .meta_module.raw
         - .meta_module.clean
       .dart:
+        - .dart.library
```

Comment:
not a huge fan of this name, it makes it sound like it has more information than it does? Also we haven't kept the `.dart` extension for other builders... with this generic of a name it kind of makes sense to keep it but a less generic name would allow you to drop it.

Reply:
do you have suggestions for alternatives? I had considered `.directives` but that naming felt a little specific to analyzer terminology to me...

Reply:
`.directives` was my initial thought but it also contains a bit more info than that (whether its any entrypoint or not).

Maybe `.module.info` or something like that?

Reply:
discussed offline. Going with `.module.library`.